.env

```diff
@@ -0,0 +1,5 @@
+PGHOST=localhost
```

Comment:
You can remove this file with `git rm .env` (this will also remove the file - I'm assuming that you have another `.env.local` file)

Reply:
It would be nice to also have an `.env.local.example` file, similar to the one in the Firebase example:

https://github.com/vercel/next.js/tree/canary/examples/with-firebase-authentication

It should have the names of the necessary configuration variables, with some placeholder values.

Eg. the one that I made looked like this:

```
PGHOST=XXXXXXXXXX
PGDATABASE=XXXXXXXXXX
PGUSERNAME=XXXXXXXXXX
PGPASSWORD=XXXXXXXXXX
```